### ISSUE TITLE
flameshot on centos 7

### ISSUE THREAD

**Issue by AlekseiSaff** [flameshot-org/flameshot#2911]
### Flameshot Version

1.12.1

### Installation Type

Compiled from source

### Operating System type and version

CentOS 7

### Description

I'm trying to run flameshot on CentOS 7, so the only available option is to compile from source with external QT5 library (since Centos7 has onlt 5.9)

so executing make with
cmake3 .. -DQt5_DIR=/usr/local/Qt-5.15.6/lib/cmake/Qt5

seems working, but after compilation I'm not able to start flameshot

flameshot says
"Failed to get information about the latest version. "Protocol \"https\" is unknown

where flameshot gui brings selection window up

### Steps to reproduce

run flameshot that was compiled from source

### System Information

centos 7 
qt 5.15.6
gcc 10.2

**Comment by AlekseiSaff** [comment]
sorry, 12.1.0

**Comment by borgmanJeremy** [comment]
Sorry in advance that this answer is going to be unsatisfactory. Your setup is pretty unique and outdated. Since I only have a limited amount of time to support Flameshot I can't really debug bespoke setup's like this. 

My gut says either OpenSSL was not linked to or the version of OpenSSL is very old.

Have you considered installing the flatpak or appimage to avoid these dependency issues?

**Comment by AlekseiSaff** [comment]
Yeah, understood. I tried from snapd, but it is not running due to old Qt. So the issue with https was really OpenSSL issue. THat one resolved, but now new error: 

Qt: Session management error: None of the authentication protocols specified are supported
